dist/licenses.txt

```diff
@@ -356,6 +356,32 @@ ACTION OF CONTRACT, NEGLIGENCE OR OTHER TORTIOUS ACTION, ARISING OUT OF OR
 IN CONNECTION WITH THE USE OR PERFORMANCE OF THIS SOFTWARE.
 
 
+moment
```

Comment:
@gdelgadot This is an unrelated change.

Reply:
yes, actually I didn't notice it

Reply:
but it has been added because of the installation of `moment.js`

Reply:
This is part of the typescript packaging.